modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -47,46 +47,32 @@ Built-in dashboards include various pages to view different types of charts:
 * JVM - Heap memory utilization
 * JVM - Thread count
 
-//* Response Time by App: Number of milliseconds (ms) required for each response.
-//* Messages Processed: Number of messages at a given time.
-//* Response Time by Inbound Endpoint: Number of milliseconds (ms) for a response from inbound endpoints to your Mule app.
-//* Response Time by Outbound Endpoint: Number of milliseconds (ms) for a response from outbound endpoints of your Mule app.
-//* CPU: Percentage of CPU used over time. The percentages are calculated per worker and as the total for all workers.
-//* Memory: Number of mebibytes (MiB) used over time.
-| *Inbound* a|
+| <<inbound-charts>> a|
 
 * Total Inbound Requests by Response Type
 * Average Response Time
 * Average Response Time by Endpoint
-** Average: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
-** Max: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 500ms; red if is greater than 500ms
-** Min: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
 * Total Requests by Endpoint
-* Response Time graphs (99, 90, 75, and 50 percentile inbound): Measured in milliseconds (ms)
+* Response Time graphs (99, 90, 75, and 50 percentile inbound)
 * Maximum Response Time
 * Total Failed Inbound Requests
-* Slow Request Count by Endpoint: Total calls in which the response time is greater than 1000 ms
-
-// I don't see Maximum Response Time or Total Failed Inbound Requests in the UI - do those exist?
-
-The following charts are available only if you are using Unified Agent and have a Titanium subscription to Anypoint Platform:
-
-* Inbound Protocol
-* Inbound Endpoint
+* Slow Request Count by Endpoint
```

Comment:
I think there's a chart skipped after "slow request...". There should be "Inbound - Maximum Response Time"

Reply:
```suggestion
* Slow Request Count by Endpoint
* Inbound - Maximum Response Time
```